example/default/json_serializable_span.dart

```diff
@@ -31,12 +36,12 @@ class JsonSerializableSpan {
 
   JsonSerializableSpan(this.span);
 
-  Map toJson() {
+  Map<String, dynamic> toJson() {
     return getFields();
   }
 
-  Map getFields() {
-    Map<String, dynamic> fieldMap = {};
+  Map<String, dynamic> getFields() {
+    Map<String, dynamic> fieldMap = <String, dynamic>{};
```

Comment:
![screen shot 2019-01-29 at 11 53 39 am](https://user-images.githubusercontent.com/2326518/51925213-91e46900-23bc-11e9-9641-5ff3cbb1bd0f.png)
How can we avoid this?

Reply:
That is from one of the lints opted into in this repo; I think that lint should be disabled unless there was a reason it was enabled.

Related: I agree with the sentiment expressed here by one of the linter package maintainers: https://github.com/dart-lang/linter/issues/1068#issuecomment-403100863 
> I do not believe that any project should enable a lint because there isn't a stated reason not to. I think, rather, that a project should only enable those lints for which there is a stated reason for it to be enabled.